sunpy/net/jsoc/jsoc.py

```diff
@@ -411,7 +404,8 @@ def request_data(self, jsoc_response, method='url', **kwargs):
             return requests[0]
         return requests
 
-    def fetch(self, jsoc_response, path=None, progress=True, overwrite=False,
+    @convert_row_to_table
+    def fetch(self, jsoc_response, *, path=None, progress=True, overwrite=False,
```

Comment:
The change to keyword only here needs a breaking changelog entry

Reply:
```suggestion
    def fetch(self, jsoc_response, path=None, progress=True, overwrite=False,
```